src/common/components/formikWrappers/Combobox.tsx

```diff
@@ -20,6 +20,8 @@ type DropdownProps = Omit<
 
 export type ComboboxProps = Omit<DropdownProps, 'toggleButtonAriaLabel'> & {
   name: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
This lintter bypass is apparently needed?

Reply:
Yep, when the `any` is used. I could try to change the `any` to some real type that the Combobox uses, but I tried for a while and didn't manage to get it to work. Didn't make much sense to fight against it.